examples/09-Incremental-Training-with-Different-Learning-Rates-and-Layer-Freezing.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>we should add the parameter return_dict=True</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>